ecf/day.sh

```diff
@@ -11,14 +12,34 @@ set -e
 echo zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz loading modules zzzzzzzzzzzzzzzzzzzzzzz
 #. /usrx/local/Modules/3.2.10/init/bash
 module purge
```

Comment:
perform `module reset` instead of `module purge`

Reply:
added to my fork that's well ahead of the EMC repo.